src/Raven.Studio/typescript/models/resources/database.ts

```diff
@@ -38,14 +38,18 @@ class database {
         this.isAdminCurrentTenant(incomingCopy.IsAdmin);
         this.name = incomingCopy.Name;
         this.disabled(incomingCopy.Disabled);
-        this.errored(!!incomingCopy.LoadError);
-
-        const nodeTag = this.clusterNodeTag();
-        const inMemberList = _.some(incomingCopy.NodesTopology.Members, x => x.NodeTag === nodeTag);
-        const inPromotableList = _.some(incomingCopy.NodesTopology.Promotables, x => x.NodeTag === nodeTag);
-        const inRehabList = _.some(incomingCopy.NodesTopology.Rehabs, x => x.NodeTag === nodeTag);
-
-        this.relevant(inMemberList || inPromotableList || inRehabList);
+        if (!!incomingCopy.LoadError) {
```

Comment:
if it has load error you don't update relevent field. Is it intendent?

Reply:
I do set `errored` to true but I do not set relevant to false as the studio would give notification that the node is not relevant for this DB, but the actual issue was that the db wasn't able to be loaded.